node/src/app/cli.rs

```diff
@@ -170,7 +170,10 @@ impl Cli {
                 // .await?;
                 // initializer2_runner.run(&mut rng).await;
 
-                initializer_runner.run(&mut rng).await;
+                let initializer_exit = initializer_runner.run(&mut rng).await;
+                if initializer_exit.is_err() {
+                    return Result::Ok(());
```

Comment:
I will add log entries.

Reply:
Actually, I can't do that since with `type RunnerExitStatus = Result<(), u8>` I can't easily add custom methods to `RunnerExitStatus` (it's a type alias for `Result` that is defined outside of my crate). I will rely on the previously logged messages.